### PR TITLE
Linter

### PR DESCRIPTION
**Pull request by weseldridge**

### CODE REVIEW

**Comment by weseldridge** [comment]
I am at a loss... It runs locally both linting and gameplay...

File: pommerman/agents/player_agent.py [hunk 1]
```diff
@@ -74,24 +74,24 @@ def __init__(self, character=characters.Bomber, agent_control='arrows'):
...
-                key.M: 6  # In Pommerman, this will freeze the game.
-            },
-            'wasd': {
-                key.W: 1,
-                key.S: 2,
-                key.A: 3,
-                key.D: 4,
-                key.E: 5,
-                key.Q: 6  # In Pommerman, this will freeze the game.
+            controls = {
```
Comment (cinjon) [inline pommerman/agents/player_agent.py]: I think this should be indented back to line up with `from pylget...`

Comment (cinjon) [reply pommerman/agents/player_agent.py]: oh actually, was it already in line with it and the `-` is causing it to not appear that way?

**Comment by cinjon** [comment]
lgtm! 👍

**Pull request merged**